www/pfSense-pkg-squid/Makefile

```diff
@@ -2,7 +2,7 @@
 
 PORTNAME=	pfSense-pkg-squid
 PORTVERSION=	0.4.44
-PORTREVISION=	9
+PORTREVISION=	10
```

Comment:
After #721 was reverted I bumped revision to 11. Please rebase your fork and bump it again. While there, please squash all commits

Reply:
done